src/group_parser.py

```diff
@@ -121,11 +121,21 @@ def detect_bursts(self, detected_signals, confirmed_signals):
 
         # Check for leftover burst that did not get logged
         if prev_symptom_tag is not None and len(burst_list) > 0:
-            burst = SymptomBurst(burst_list)
+            burst = symptom_burst.SymptomBurst(burst_list)
             self.bursts.append(burst)
 
             if prev_symptom_tag not in self.burst_dict:
                 self.burst_dict[prev_symptom_tag] = [burst]
             else:
                 self.burst_dict[prev_symptom_tag].append(burst)
 
+    def convert_self_to_dict(self):
```

Comment:
The "self" in the name is probably not that helpful; a convert_to_dict method with no parameters would generally be implied to be operating on self.

Reply:
Ok - I can change the function name.